presto-main/src/main/java/com/facebook/presto/server/security/ServerSecurityModule.java

```diff
@@ -66,6 +66,10 @@ private void setupLdap(LdapServerConfig ldapServerConfig, Binder binder)
             checkState(ldapServerConfig.getUserObjectClass() != null, "Missing property 'authentication.ldap.user-object-class'");
         }
 
+        if (ldapServerConfig.getLdapUrl().startsWith("ldap://")) {
```

Comment:
With `ldap.url=ldapserver:389`, it gives `MalformedURLException`.
With `ldap.url=ldaps://ldapserver:389`, it gives `SSLHandshakeException`.